h/nipsa/services.py

```diff
@@ -15,47 +15,41 @@ def __init__(self, session):
         self.session = session
 
     @property
-    def flagged_userids(self):
+    def flagged_users(self):
         """
-        A list of all the NIPSA'd user IDs.
+        A list of all the NIPSA'd users.
 
         :rtype: list of unicode strings
         """
-        return [u.userid for u in self.session.query(models.NipsaUser)]
+        return self.session.query(User).filter_by(nipsa=True)
```

Comment:
This will be quite a bit slower than what it was before, but it's probably alright since we don't call this method very often. New Relic will help us identify this once it becomes an issue.